pkg/apb/watch_pod.go

```diff
@@ -55,6 +57,18 @@ func watchPod(podName string, namespace string) error {
 			if errorPullingImage(podStatus.ContainerStatuses) {
 				return ErrorPodPullErr
 			}
+
+			// handle the return code from the pod
+			exitCode := getExitCode(podStatus.ContainerStatuses)
+			if exitCode == 8 {
+				log.Errorf("Pod [ %s ] failed - action not found.", podName)
```

Comment:
There might be a way to make this more informative. The caller of watchPod is one of the actions (bind, undbind, provision, deprovision).  So the caller knows what action was performed.  Inside watchPod, we know the caller's action failed. We don't know what it is, but we do know the playbook for that action doesn't exist if the exitCode is 8.

A better way to do this might be to return something like ```Pod [ %s ] failed - the action's playbook was not found```. Then at the action level, we change ```log.Errorf("APB Execution failed - %v", err)``` to identify the action and bubble up the error.  For bind, this could look like  ```log.Errorf("Bind action failed - %v", err)```

What do folks think?

Reply:
This sounds reasonable to me. More context for the end user the better. I would actually suggest moving to https://github.com/pkg/errors for this kind of thing. It is very useful.
Has a method
```
errors.Wrap(err,"some additional context")
```
that returns an error. It also provides the benefit of providing stack traces when the errors are logged with ```%+v```

Reply:
@maleck13 I think we should discuss moving to pkg/errors in a proposal I have created to #717 track this change. I agree that this package does help in debugging errors and is super helpful.

Reply:
It sounds great, @shawn-hurley it does sound like a broader convention we should adopt and talk about how we want to get there.